app/models/core/event.rb

```diff
@@ -0,0 +1,5 @@
+class Core::Event < ApplicationRecord
+     belongs_to :location, optional: true
+     validates :title, presence: :true
+     :location_id
```

Comment:
Didn't we remove this? Just putting there here is like placing a string on a line of code.
Was this potentially in a different branch?